PinpointKit/PinpointKit/Sources/AnnotationView.swift

```diff
@@ -8,18 +8,16 @@
 
 import UIKit
 
+/// The alpha value used for annotation borders.
+let BorderAlpha: CGFloat = 0.7
+
 /// The base annotation `UIView` subclass.
-public class AnnotationView: UIView {
-    
-    /// The alpha value used for annotation borders.
-    static let BorderAlpha: CGFloat = 0.7
+public protocol AnnotationView {
```

Comment:
> Since this protocol no longer necessitates implementation on just UIView objects, but instead has a view object, 

Good point

> AnnotationControlling,AnnotationBehaviorControlling,AnnotationAppearanceControlling`,

What about `Annotation`?